pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -660,6 +660,19 @@ func (dm *DockerManager) runContainer(
 		SecurityOpt: securityOpts,
 	}
 
+	// Set sysctls if requested
+	sysctls, err := api.SysctlsFromPodAnnotation(pod.Annotations[api.SysctlsPodAnnotationKey])
```

Comment:
Thanks for the heads up. I am also counting on the node e2e tests to let me know what feature is missing in the new shim.